StructureDefinition/Extension-England-ODSOrganizationRole.xml

```diff
@@ -0,0 +1,99 @@
+<?xml version="1.0" encoding="utf-8"?>
+<StructureDefinition xmlns="http://hl7.org/fhir">
+  <id value="Extension-England-ODSOrganizationRole" />
+  <url value="https://fhir.nhs.uk/StructureDefinition/Extension-England-ODSOrganizationRole" />
+  <version value="1.0.0" />
+  <name value="ExtensionEnglandODSOrganizationRole" />
+  <title value="Extension England ODS Organization Role" />
+  <status value="draft" />
+  <date value="2023-06-01" />
+  <publisher value="NHS England" />
+  <contact>
+    <name value="NHS England" />
+    <telecom>
+      <system value="email" />
+      <value value="interoperabilityteam@nhs.net" />
+      <use value="work" />
+	  <rank value="1" />
+    </telecom>
+  </contact>
+  <description value="An Extension to describe what an Organization does and whether this is the primary role of the Organization." />
+  <purpose value="An Extension to describe what an Organization does and whether this is the primary role of the Organization." />
```

Comment:
I will say the rule of no repetition in the NHSE CTA Dev. and guide may not allow duplicating content of description and purpose. Better to leave out purpose